app/src/main/java/com/example/learn09082021v001/MainActivity.kt

```diff
@@ -37,39 +40,47 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        // view config
+        viewMainAndroid.edtName.isEnabled = false
+        viewMainIos.edtName.isEnabled = false
+        viewMainAndroid.btnDEC.visibility = View.GONE
```

Comment:
write a function to gone in the custom view instead of call child view gone separately
e.g.
fun goneButtonAction(){
btnDEC.visibility = View.GONE
btnINC.visibility = View.GONE
edtName.isEnabled = false
}

in here just need to call 
viewMainAndroid. goneButtonAction
viewMainIos. goneButtonAction